lnclient/ldk/ldk.go

```diff
@@ -1939,3 +1924,23 @@ func (ls *LDKService) ConsumeEvent(ctx context.Context, event *events.Event, glo
 		ls.backupChannels()
 	}
 }
+
+// checkLDKErr is a helper function to convert errors returned from LDK to
```

Comment:
@im-adithya FYI this was a breaking change in the uniffi-bindgen-go release

Reply:
Got it, thanks for the heads up! 🙌